src/Approximations/template_directions.jl

```diff
@@ -349,7 +431,6 @@ Spherical directions representation.
 
 ### Fields
 
-- `Nθ`    -- length of the partition of the azimuthal angle
```

Comment:
Why did you remove this line?

Reply:
if i removed `Nθ` in Spherical directions it was unintentional, probably with the merge conflict.. thx